src/module/account/settings/setting-form.component.test.tsx

```diff
@@ -0,0 +1,57 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import SettingForm from './setting-form.component';
+
+function renderSettingPage() {
+  render(<SettingForm />);
+}
+
+describe('setting', () => {
```

Comment:
shouldn't there be a test for happy path?

Reply:
Add a todo reminder here